contracts/prize-strategy/PrizeSplit.sol

```diff
@@ -132,8 +132,7 @@ abstract contract PrizeSplit is IPrizeSplit, Ownable {
         uint256 prizeSplitsLength = _prizeSplits.length;
 
         for (uint8 index = 0; index < prizeSplitsLength; index++) {
-            PrizeSplitConfig memory split = _prizeSplits[index];
-            _tempTotalPercentage = _tempTotalPercentage + split.percentage;
+            _tempTotalPercentage = _tempTotalPercentage + _prizeSplits[index].percentage;
```

Comment:
```suggestion
            _tempTotalPercentage += _prizeSplits[index].percentage;
```

Reply:
I'm actually not sure this is cheaper since `_prizeSplits` is in storage, so each time we access `_prizeSplits[index].percentage`, it will cost us 200 in gas.

Reply:
either way it's one word